### ISSUE TITLE
page.EvaluateHandler typings for 1.19.0 are different than docs

### ISSUE THREAD

**Issue by angrykoala** [puppeteer/puppeteer#4763]
Steps to reproduce
Update `@types/puppeteer` to 1.19.0 

**Tell us about your environment:**

* Puppeteer version: 1.19.0
* Platform / OS version: Linux
* URLs (if applicable):
* Node.js version: 10.16.0

According to puppeteer docs for v1.19.0, the evaluateHandle method support both, a callback function or a string:
**[page.evaluateHandle](https://pptr.dev/#?product=Puppeteer&version=v1.19.0&show=api-pageevaluatehandlepagefunction-args)**

However, since I updated `@types/puppeteer` to v1.19.0 I'm getting the following error 
```
TS2345: Argument of type 'EvaluateFn' is not assignable to parameter of type '(...args: any[]) => any'.
  Type 'string' is not assignable to type '(...args: any[]) => any'.
```

Which is silenced by passing only a callback instead of `string | ((...args: any[]) => any))`

**Comment by SimonSchick** [comment]
Probably my bad, I was trying to consolidate these things and ended up breaking this, will fix asap :)

**Comment by rajshrimohanks** [comment]
This is still not fixed.

**Comment by angrykoala** [comment]
This is fixed in @types/puppeteer 1.20.2
